pldoc/_components/breadcrumbs.md

```diff
@@ -0,0 +1,54 @@
+---
+layout:                 pattern
+title:                  Breadcrumbs
+date:                   2016-02-29 00:00:00
+
+categories:             component
+tags:
+- atomic
+- pattern
+- js
+- breadcrumbs
+- nav
+
+slug:                   breadcrumbs
+
+url_styles:             "patterns/_breadcrumbs.scss"
+
+description:            Breadcrumb navigation.
+
+info:                   Accessible tiered navigation using breadcrumbs.
+ui-toolkit-component:   breadcrumbs
+js:                     "/examples/breadcrumbs-js.html"
+---
+<div class="breadcrumbs-basic example-set">
+  <h2>Basic breadcrumbs</h2>
+  <header class="has-breadcrumbs hd-3"></header>
+  <a class="add-breadcrumb" href="">Add breadcrumb</a>
+</div>
+
+<div class="breadcrumbs-prerendered example-set">
+  <h2>Pre-rendered breadcrumbs</h2>
+  <header class="has-breadcrumbs hd-3">
+    <div class="sr-is-focusable" tabindex="-1"></div>
```

Comment:
I actually have no idea - this is direct from the Teams implementation of breadcrumbs.

Just asked Chris, he says it's not necessary, so I'll remove it. (Requires doing a UITK patch release, unfortunately.)